javascripts/discourse/routes/user-portfolio.js

```diff
@@ -17,20 +15,21 @@ export default DiscourseRoute.extend({
     });
   },
 
-  model: function () {
+  model() {
     const filterParams = {};
+    const filterTags = settings.portfolio_tags.replace(/\|/g, "+");
+    let tagQuery = filterTags ? `?tags=${filterTags}+` : "";
```

Comment:
```suggestion
    const tagQuery = filterTags ? `?tags=${filterTags}+` : "";
```

since it isn't changed after this